driver/ppm_flag_helpers.h

```diff
@@ -1881,6 +1881,16 @@ static __always_inline u32 dup3_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+static __always_inline uint32_t epoll_create1_flags_to_scap(uint32_t flags)
+{
+	uint32_t res = 0;
+#ifdef EPOLL_CLOEXEC
```

Comment:
we should add also this flag in the missing_definitions.h file otherwise the modern probe doesn't have the definition of this flag. this for example could be a good place :point_down: 
https://github.com/falcosecurity/libs/blob/3d5e6e2c53f4b2dd4bc7c8ad48d45632e10b6e05/driver/modern_bpf/definitions/missing_definitions.h#L661

We can add something like 

```c
//////////////////////////
// epoll_create1 flags
//////////////////////////

#define EPOLL_CLOEXEC ...

```